README.md

```
@@ -34,114 +34,94 @@
+- `list` - List stacks on a host
+- `deploy` - Deploy a stack
+- `up` - Manage stack lifecycle (bring up)
+- `down` - Manage stack lifecycle (bring down)
+- `restart` - Manage stack lifecycle (restart)
+- `build` - Manage stack lifecycle (build)
+- `discover` - Discover compose paths on a host
+- `logs` - Get stack logs
+- `migrate` - Migrate stack between hosts
+
+**Natural language examples:**
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)